lib/signs_ui/signs/messages.ex

```diff
@@ -63,19 +63,31 @@ defmodule SignsUi.Signs.Messages do
   defp parse_commands(commands) do
     commands
     |> Enum.map(fn command ->
-      [duration, zone_line, text] = String.split(command, ["~", "-"])
+      [duration, zone_line_text] = String.split(command, ["~"])
       duration_regex = ~r/([a-z])([\d]+)/
       [_original, _expires?, duration] = Regex.run(duration_regex, duration)
 
-      text =
-        text
-        |> URI.decode()
-        |> String.trim("\"")
-        |> String.replace("+", " ")
+      [zone_line | text_list] = String.split(zone_line_text, ["-"])
+
+      text_list =
+        text_list
+        |> Enum.map(fn text ->
+          text_time_regex = ~r/([\w\d\s]+).?([\d]?+)$/
```

Comment:
Right before the digit at the end, should that be a literal period `\.` maybe, rather than a wildcard?

Reply:
oh, yeah -  good catch